src/renderer/components/server-modal-form.jsx

```diff
@@ -61,7 +61,7 @@ export default class ServerModalForm extends Component {
       onUnchecked: () => this.setState({ ssl: false }),
     });
 
-    $(this.refs.privateKeyWithPassphrase).checkbox({
+    $(this.refs && this.refs.privateKeyWithPassphrase).checkbox({
```

Comment:
Can the refs be null or undefined at some time? Well, seems it is not related to this issue. If you could move it with the other one I suggested before would give us a better tracking in the repository history.